charts/scalardb-analytics-postgresql/templates/scalardb-analytics-postgresql/deployment.yaml

```diff
@@ -0,0 +1,155 @@
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.scalardbAnalyticsPostgreSQL.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.scalardbAnalyticsPostgreSQL.securityContext | nindent 12 }}
+          image: "{{ .Values.scalardbAnalyticsPostgreSQL.image.repository }}:{{ .Values.scalardbAnalyticsPostgreSQL.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.scalardbAnalyticsPostgreSQL.image.pullPolicy }}
+          ports:
+            - containerPort: 5432
+          env:
+          - name: POSTGRES_DB
+            value: "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+          envFrom:
+          - secretRef:
+              name:  "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.secretName }}"
+          {{- if .Values.scalardbAnalyticsPostgreSQL.secretName }}
+          - secretRef:
+              name: "{{ .Values.scalardbAnalyticsPostgreSQL.secretName }}"
+          {{- end }}
+          volumeMounts:
+            - mountPath: /etc/database.properties
+              name: database-properties-volume
+              subPath: database.properties
+          resources:
+            {{- toYaml .Values.scalardbAnalyticsPostgreSQL.resources | nindent 12 }}
+          startupProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 60
+            periodSeconds: 5
+          livenessProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+        - name: schema-importer
+          securityContext:
+            {{- toYaml .Values.schemaImporter.securityContext | nindent 12 }}
+          image: "{{ .Values.schemaImporter.image.repository }}:{{ .Values.schemaImporter.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.schemaImporter.image.pullPolicy }}
+          command: [/entrypoint.sh]
+          args:
+            - java
+            - -jar
+            - /app.jar
+            - import
+            - --host
+            - localhost
+            - --port
+            - "5432"
+            - --database
+            - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+            - --user
+            - postgres
+            - --password
+            - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.password }}"
```

Comment:
@superbrothers (cc: @choplin )

Thank you for your suggestion! It's a very important point!
In conclusion, we can remove this `--password` option. So, I just removed it in ec0c4b5a554b99a15c0b0a6e894becb04b1ec97c.

After I checked this point, I found that we don't need to set the `--password` option if we run `Schema Importer` against `localhost`. This is because the default `pg_hba.conf` allows connections from `localhost` (i.e., connections from the sidecar container via `localhost`) without a password as follows.

```console
host    all             all             127.0.0.1/32            trust
```

Note: Of course, the connections from outside of the pod still need password authentication.

So, we can remove this `--password` option if we run `Schema Importer` in the sidecar container.

---

For your information, we will update `Schema Importer` to get a password from an environment variable in the future version.
https://github.com/scalar-labs/scalardb-analytics-postgresql/issues/47

Also, there is a possibility that the default value of pg_hba.conf will be changed. However, even in such cases, we can configure pg_hba.conf file ourselves because the PostgreSQL container image provided such feature `-c 'config_file=/etc/postgresql/postgresql.conf'`.